xmtp_mls/src/groups/members.rs

```diff
@@ -49,28 +49,25 @@ impl MlsGroup {
         let members = association_state_map
             .into_iter()
             .map(|association_state| {
-                let is_admin = self
-                    .is_admin(association_state.inbox_id().to_string())
-                    .unwrap();
-                let is_super_admin = self
-                    .is_super_admin(association_state.inbox_id().to_string())
-                    .unwrap();
+                let inbox_id_str = association_state.inbox_id().to_string();
+                let is_admin = self.is_admin(inbox_id_str.clone())?;
```

Comment:
This is going to load the MLS group from the DB on every call, isn't it?

Think we can just load the mutable metadata at the top and use a single instance of it for all the checks.

Reply:
good catch, fixed that here: f3c97a0b19a601536456ae53850406c5e18bc401